docs/upgrading.md

```diff
@@ -1,5 +1,118 @@
 # Upgrading
 
+## To 0.7x
+
+### Accept and Verify JSON Web Tokens
+
+We made changes to API authentication. Access to the API is now authenticated, and
+users can provide a URI to the keyset used to sign the JWT bearer tokens. Users can
+also provide a JSON pointer into the JWT claims for the Chronicle identity. A Chronicle
+`AuthId` with a verified external ID can then be found in the GraphQL context.
+
+We also converted the `JWT` `AuthId` variant to be a structure of both a json-pointer
+-derived scalar and JWT claims.
+
+### Anonymous Identity Variant
+
+By default, Chronicle allows unauthenticated access to the GraphQL API, classifying
+such a user as "anonymous." Adds an identity variant emanating from GraphQL or CLI,
+not authorized via Auth0, allowing for the option to pass a transactor key as a CLI
```

Comment:
No need to mention Auth0 specifically, deployments may be using other providers.